src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -10,9 +10,50 @@ namespace Microsoft.ML.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        /// Converts the model to ONNX format.
+        ///
+        /// This API converts the model to ONNX format by inspecting the transform pipeline 
+        /// from the end, checking for components that know how to save themselves as ONNX. 
+        /// The first data view in the transform pipeline that does not know how to save itself 
```

Comment:
>data view [](start = 22, length = 9)

While data view is a central concept in the underlying implementation, I don't think that the current API actually exposes this concept in any meaningful fashion, does it? Which means that this is an undefined concept, from the point of view of an API user.

Reply:
Maybe just "item" for now, since from context it is clear that you're talking about transforms.

---
In reply to: [198256303](https://github.com/dotnet/machinelearning/pull/419#discussion_r198256303) [](ancestors = 198256303)